2024-03-28 15:25:56 INFO [32mpytest tests/frontend/test_e2e_project.py::TestProjectE2E::test_e2e_create_project[chromium] (setup)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:*/*'
-H 'Connection:keep-alive'
-H 'Authorization:Basic OjY1MzMyMTY2NzQ2NzU1Mjk1NDY=' 
-d ''
2024-03-28 15:26:09 INFO [32mpytest tests/frontend/test_e2e_project.py::TestProjectE2E::test_e2e_create_project[chromium] (teardown)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:28432376-964e-44ae-843d-87eb046cafac'
-H 'Cookie:TCSESSIONID=8B50E3DBA3FE200C8DB6BC69C10BAE27'
-H 'Authorization:Basic OjY1MzMyMTY2NzQ2NzU1Mjk1NDY=' 
-d '' 
 

2024-03-28 15:26:09 INFO [32mpytest tests/frontend/test_e2e_project.py::TestProjectE2E::test_e2e_create_project[chromium] (teardown)[0m
curl - X DELETE 'http://localhost:8111/app/rest/projects/id:HU5GRpnAT8T' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:28432376-964e-44ae-843d-87eb046cafac'
-H 'Cookie:TCSESSIONID=8B50E3DBA3FE200C8DB6BC69C10BAE27'
-H 'Content-Length:0'
-H 'Authorization:Basic OjY1MzMyMTY2NzQ2NzU1Mjk1NDY=' 
-d '' 
 

2024-03-28 15:26:09 INFO 	RESPONSE:
STATUS_CODE: [31m404[0m
DATA: [31mResponding with error, status code: 404 (Not Found).
Details: jetbrains.buildServer.server.rest.errors.NotFoundException: No project found by locator 'count:1,id:HU5GRpnAT8T'. Project cannot be found by external id 'HU5GRpnAT8T'.
Could not find the entity requested. Check the reference is correct and the user has permissions to access the entity.[0m